etc/buildsys/rules.mk

```diff
@@ -275,13 +275,19 @@ $(foreach MS,$(MANPAGE_SECTIONS),$(MANDIR)/man$(MS)/%.$(MS)): %.txt
 	fi
 
 # execute every test in $(BINS_test)
```

Comment:
Minor suggestion, $(BINS_test) does not exist anymore, so the comment could be updated
```suggestion
# execute every test in $(BINS_gtest) and $(BINS_catch2test)
```